src/components/App/OutputSection.tsx

```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link';
+import React from 'react';
+import { AiOutlineArrowDown } from 'react-icons/ai';
+import { FaPlus } from 'react-icons/fa';
+import { IoIosCopy } from 'react-icons/io';
+import { IoIosShareAlt } from 'react-icons/io';
+
+import Button from '@/components/Button';
+
+interface OutputSectionProps {
+    originalUrl: string;
+    shortUrl: string;
+    handleCreateNew: () => void;
+    handleCopyUrl: () => void;
+}
+
+const OutputSection: React.FC<OutputSectionProps> = ({ shortUrl, originalUrl, handleCopyUrl, handleCreateNew }) => (
+    <section className="flex flex-col   justify-between items-center space-y-0 space-x-0 rounded-2xl mt-5 sm:mt-10 w-[100%] text-gray-400">
+        <h1 className="text-3xl md:text-4xl xl:text-4xl text-center mb-8 text-white font-semibold">
+            Your Tiny URL is ready! 🎉🎉
+        </h1>
+        <span className="ml-2 p-4 text-center w-full sm:w-[80%] ellipsis overflow-hidden overflow-ellipsis whitespace-nowrap">
+            {originalUrl}
+        </span>
+        <AiOutlineArrowDown style={{ fontSize: '3rem' }} />
+
+        <div
+            className="text-white flex flex-col md:flex-row justify-center items-center space-y-2 sm:space-y-0 space-x-0 sm:space-x-2 rounded-2xl w-full  lg:w-[80%] xl:w-[80%] cursor-pointer bg-black p-1"
+            onClick={handleCopyUrl}
+        >
+            <span className="ml-2 p-4 text-center w-full sm:w-[80%] sm:text-2xl md:text-3xl xl:text-3xl">
+                {shortUrl.replace(/(^\w+:|^)\/\//, '')}
```

Comment:
can you move this to constant

Reply:
No, we cannot move this to constants. Since 'replace' is the function, creating a separate utils function is not a good idea. Therefore, we can keep this as is.